### PR TITLE
fix broken link of kube-apiserver

### PR DESCRIPTION
**Pull request by zhanghuidinah**
fix broken link of kube-apiserver

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ answer the following questions:
 
 {{% capture body %}}
 
-[Kube-apiserver][kube-apiserver] performs auditing. Each request on each stage
+[Kube-apiserver](#https://github.com/kubernetes/website/blob/release-1.12/content/zh/docs/admin/kube-apiserver.md) performs auditing. Each request on each stage
```
Comment (chenrui333) [inline content/en/docs/tasks/debug-application-cluster/audit.md]: why you need to have `#` for the hyperlink? Also, it seems not working properly.
https://5c0a38fbe470857ab734c55b--k8s-v1-12.netlify.com/docs/tasks/debug-application-cluster/audit/

Comment (chenrui333) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: On the other side, if you check with this link, k8s-v1-12.netlify.com/docs/tasks/debug-application-cluster/audit/
it seems it works just fine.

**Comment by chenrui333** [comment]
@zhanghuidinah @markthink 需要实际对着 preview build 测一下这个 PR， 我看原文 好像没有问题。

File: content/en/docs/tasks/debug-application-cluster/audit.md (hunk 1 above)
Comment (zhanghuidinah) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: Sorry, cause for my local markdown language needs to add `#` to make the hyperlink take effect. And I will check this pr again.

Comment (chenrui333) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: hyperlink 加`#` 不太符合markdown 的syntax，所以才有之前的疑问。

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ answer the following questions:
 
 {{% capture body %}}
 
-[Kube-apiserver][kube-apiserver] performs auditing. Each request on each stage
+[Kube-apiserver](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) performs auditing. Each request on each stage
```
Comment (tengqm) [inline content/en/docs/tasks/debug-application-cluster/audit.md]: this change is incorrect. The link address is found in line 363. The original syntax is correct.

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 3]
```diff
@@ -327,7 +327,7 @@ different users into different files.
     $ bin/logstash -f /etc/logstash/config --path.settings /etc/logstash/
     ```
 
-1. create a [kubeconfig file](/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/) for kube-apiserver webhook audit backend
+1. create a [kubeconfig file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) for kube-apiserver webhook audit backend
```
Comment (tengqm) [inline content/en/docs/tasks/debug-application-cluster/audit.md]: This change is incorrect. We use relative links rather than absolute links.

**Comment by zparnold** [comment]
@zhanghuidinah have you had a chance to address @tengqm 's comments?

**Comment by zacharysarah** [comment]
@zhanghuidinah Please feel free to reopen when you're ready to address feedback.

**Pull request closed without merging**